From a805071445caa85898faad24ed236c030f482a8c Mon Sep 17 00:00:00 2001
From: Evan Quan <evan.quan@amd.com>
Date: Fri, 27 Oct 2017 12:12:39 +0800
Subject: [PATCH 3267/5855] drm/amd/powerplay: fix compile errors

It's brought by "drm/amd/powerplay: fix performance drop on Vega10"
since smumgr layer has not been removed yet on current branch.

Change-Id: I4e5b79ab99b27bb7bb89a77a7aa534f90495119f
Signed-off-by: Evan Quan <evan.quan@amd.com>
Reviewed-by: Ken Wang <ken.wang@amd.com>
Reviewed-by: Junwei Zhang <Jerry.Zhang@amd.com>
---
 drivers/gpu/drm/amd/powerplay/hwmgr/vega10_hwmgr.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/powerplay/hwmgr/vega10_hwmgr.c b/drivers/gpu/drm/amd/powerplay/hwmgr/vega10_hwmgr.c
index 64e459e..f9099de 100644
--- a/drivers/gpu/drm/amd/powerplay/hwmgr/vega10_hwmgr.c
+++ b/drivers/gpu/drm/amd/powerplay/hwmgr/vega10_hwmgr.c
@@ -2881,7 +2881,7 @@ static int vega10_enable_dpm_tasks(struct pp_hwmgr *hwmgr)
 
 	if ((data->smu_version == 0x001c2c00) ||
 			(data->smu_version == 0x001c2d00)) {
-		tmp_result = smum_send_msg_to_smc_with_parameter(hwmgr,
+		tmp_result = smum_send_msg_to_smc_with_parameter(hwmgr->smumgr,
 				PPSMC_MSG_UpdatePkgPwrPidAlpha, 1);
 		PP_ASSERT_WITH_CODE(!tmp_result,
 				"Failed to set package power PID!",
-- 
2.7.4

